.rubocop.yml

```diff
@@ -1237,7 +1237,7 @@ RSpec/MessageSpies:
   Enabled: false
 
 RSpec/MultipleExpectations:
```

Comment:
Maybe it would be better to just disable this cop

Reply:
I meant for the whole project, instead of `Max: 17` just `Enabled: false`, it doesn't add much value

Reply:
Ahh okay. I understood it now. I'll change it now